pkg/config/config.go

```diff
@@ -1581,6 +1575,10 @@ func (r *RuntimeHandler) Validate(name string) error {
 	if err := r.ValidateRuntimeAllowedAnnotations(); err != nil {
 		return err
 	}
+	if err := r.ValidateContainerMinMemory(name); err != nil {
+		logrus.Errorf("Unable to set minimum container memory for runtime handler %q: %v", name, err)
```

Comment:
Should we return the error here?

Reply:
we fallback to the default min memory so the config ends up valid, which was the original author's intent

Reply:
:+1: